dvc/command/version.py

```diff
@@ -1,29 +1,62 @@
 from __future__ import unicode_literals
 
+import os
+import psutil
 import platform
 import argparse
 import logging
 
+from dvc.repo import Repo
 from dvc.command.base import CmdBaseNoRepo, append_doc_link
 from dvc.version import __version__
+from dvc.exceptions import NotDvcRepoError
 
 
 logger = logging.getLogger(__name__)
 
 
 class CmdVersion(CmdBaseNoRepo):
     def run(self):
-        info = (
-            "DVC version: {dvc_version}\n"
-            "Python version: {python_version}\n"
-            "Platform: {platform_type}"
-        ).format(
-            dvc_version=__version__,
-            python_version=platform.python_version(),
-            platform_type=platform.platform(),
-        )
-        logger.info(info)
-        return 0
+        try:
+            root_directory = Repo.find_root()
+
+            info = (
+                "DVC version: {dvc_version}\n"
+                "Python version: {python_version}\n"
+                "Platform: {platform_type}\n"
+                "Filesystem Type: {filesystem_type}\n"
+                "Symlink: {symlink}"
+            ).format(
+                dvc_version=__version__,
+                python_version=platform.python_version(),
+                platform_type=platform.platform(),
+                filesystem_type=self.get_fs_type(
+                    os.path.abspath(root_directory)
+                ),
+                symlink=os.path.islink(os.path.abspath(root_directory)),
+            )
+            logger.info(info)
+            return 0
+
+        except NotDvcRepoError:
```

Comment:
This exception is covered in cli.py, no need to catch it here again. Btw, your command can work and is useful both inside and outside of a repo. Just don't put repo-related info if the command is run outside of a dvc repo.